src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -584,11 +578,17 @@ function discoverController(
 
 
   function beginSegmentedFetch() {
+    $scope.fetchError = undefined;
+
     $scope.searchSource.onBeginSegmentedFetch(initSegmentedFetch)
       .catch((error) => {
-        notify.error(error);
-        // Restart.
-        beginSegmentedFetch();
```

Comment:
Note that I've removed this. Everything still seems to be working though. @Bargs or @lukasolson do you think this is OK to remove?

Reply:
@cjcenizal did you test whether auto refresh continues to work after an error response? If I had to guess the purpose of this line, that would be it. I didn't write this though, so I don't know for sure. Personally, if I don't know the purpose of a line and I don't have a compelling reason to change it, I generally leave it alone.